x-pack/filebeat/module/okta/system/test/okta-system-test.json.log-expected.json

```diff
@@ -0,0 +1,251 @@
+[
+    {
+        "@timestamp": "2020-02-14T22:18:51.843Z",
+        "client.as.number": null,
+        "client.as.organization.name": null,
+        "client.domain": null,
+        "client.geo.city_name": "Dublin",
+        "client.geo.country_name": "United States",
+        "client.geo.location.lat": 37.7201,
+        "client.geo.location.lon": -121.919,
+        "client.geo.region_name": "California",
+        "client.ip": "108.255.197.247",
```

Comment:
should probably be in client.address as well.  Also the ip should be be copied to related.ip